data/plugins/language_c.lua

```diff
@@ -65,12 +78,6 @@ syntax.add {
     { pattern = "__[%u%l]+__",           type = "number"   },
     -- all other functions
     { pattern = "[%a_][%w_]*%f[(]",      type = "function" },
```

Comment:
Now that you are working on this maybe also change the functions pattern to allow spaces between name and parenthesis with something like:
```lua
{ pattern = "[%a_][%w_]*()%s*%f[(]", type = { "function", "operator" } },
```
should work on language_cpp too